src/main/java/taskat2/Human.java

```diff
@@ -0,0 +1,61 @@
+package taskat2;
+
+public class Human {
+    private String name;
+    private String eyeColor;
+    private int age;
+    private int height;
+
+    public Human(String name, String eyeColor, int age, int height) {
+        this.name = name;
+        this.eyeColor = eyeColor;
+        this.age = age;
+        this.height = height;
+    }
+
+    public Human() {
+        this.name = "Gregory";
+        this.eyeColor = "Dark";
+        this.age = 50;
+        this.height = 190;
+
+    }
```

Comment:
so if you would call constructor without parameters it would always creates you the same hardcoded Human instance? what is the reason of it? what for?